internal/impl/openai/transcription_processor.go

```
@@ -124,6 +139,30 @@ func (p *transcriptionProcessor) Process(ctx context.Context, msg *service.Messa
 		}
 		body.Prompt = pr
 	}
+
+	if p.format != nil {
+		t, err := p.format.TryString(msg)
+		if err != nil {
+			return nil, fmt.Errorf("%s interpolation error: %w", otspFieldFormat, err)
+		}
+
+		var format oai.AudioResponseFormat
+		switch strings.ToLower(t) {
+		case "verbose_json":
+			format = oai.AudioResponseFormatVerboseJSON
+		case "text":
+			format = oai.AudioResponseFormatText
+		case "vtt":
+			format = oai.AudioResponseFormatVTT
+		case "srt":
+			format = oai.AudioResponseFormatSRT
+		default:
+			format = oai.AudioResponseFormatJSON
```

Consider returning an error instead of silently discarding invalid formats.